Previously mypy was complaining on this file:
```
$ mypy code-format-helper.py 
code-format-helper.py:29: error: Bracketed expression "[...]" is not valid as a type  [valid-type]
code-format-helper.py:29: note: Did you mean "List[...]"?
code-format-helper.py:36: error: "FormatHelper" has no attribute "friendly_name"  [attr-defined]
code-format-helper.py:44: error: "FormatHelper" has no attribute "instructions"  [attr-defined]
code-format-helper.py:90: error: "FormatHelper" has no attribute "friendly_name"  [attr-defined]
code-format-helper.py:94: error: Bracketed expression "[...]" is not valid as a type  [valid-type]
code-format-helper.py:94: note: Did you mean "List[...]"?
code-format-helper.py:123: error: Bracketed expression "[...]" is not valid as a type  [valid-type]
code-format-helper.py:123: note: Did you mean "List[...]"?
code-format-helper.py:132: error: Bracketed expression "[...]" is not valid as a type  [valid-type]
code-format-helper.py:132: note: Did you mean "List[...]"?
code-format-helper.py:135: error: Return value expected  [return-value]
code-format-helper.py:162: error: Bracketed expression "[...]" is not valid as a type  [valid-type]
code-format-helper.py:162: note: Did you mean "List[...]"?
code-format-helper.py:171: error: Bracketed expression "[...]" is not valid as a type  [valid-type]
code-format-helper.py:171: note: Did you mean "List[...]"?
code-format-helper.py:174: error: Return value expected  [return-value]
Found 11 errors in 1 file (checked 1 source file)
```

--- a/llvm/utils/git/code-format-helper.py
+++ b/llvm/utils/git/code-format-helper.py
@@ -165,7 +165,7 @@ class DarkerFormatHelper(FormatHelper):
     friendly_name = "Python code formatter"
 
     @property
-    def instructions(self):
+    def instructions(self) -> str:
         return " ".join(self.darker_cmd)
 
     def filter_changed_files(self, changed_files: list[str]) -> list[str]:

